static/src/javascripts/projects/commercial/modules/third-party-tags/plista.js

```diff
@@ -3,13 +3,16 @@ import fastdom from 'fastdom';
 import $ from 'lib/$';
 import config from 'lib/config';
 import detect from 'lib/detect';
-import template from 'lodash/utilities/template';
 import trackAdRender from 'commercial/modules/dfp/track-ad-render';
 import { commercialFeatures } from 'commercial/modules/commercial-features';
-import plistaStr from 'raw-loader!commercial/views/plista.html';
 import { loadScript } from 'lib/load-script';
 
-const plistaTpl = template(plistaStr);
```

Comment:
do we proscribe `lodash/utilities/template` in the lint config? we probably should?